frontend_server/lib/server.dart

```diff
@@ -96,14 +96,6 @@ abstract class CompilerInterface {
   /// taking into account some changed(invalidated) sources.
   Future<Null> recompileDelta();
 
-  /// Accept results of previous compilation so that next recompilation cycle
-  /// won't recompile sources that were previously reported as changed.
-  void acceptLastDelta();
```

Comment:
With these gonedDoes the `resetIncrementalCompiler` below still makes sense?

Reply:
I have implemented `resetIncrementalCompiler` now.